packages/ui/src/components/DateInput/index.tsx

```diff
@@ -204,6 +238,17 @@ export const DateInput = ({
     registerLocale(localeCode, locale)
   }
 
+  const valueStart = `${
+    startDate !== undefined && startDate !== null ? format(startDate) : ''
+  }`
+  const valueEnd = `${
+    endDate !== undefined && endDate !== null ? format(endDate) : ''
+  }`
+
+  const valueFormat = selectRange
+    ? `${valueStart} - ${valueEnd}`
+    : format(value)
```

Comment:
Here you need to check if `valueStart` and `valueEnd` exists or if they don't we have an input containing only `-`

<img width="989" alt="Screenshot 2023-10-12 at 10 19 29" src="https://github.com/scaleway/ultraviolet/assets/15812968/46f1668f-0f88-4739-bade-b84c6fd0f723">